ui/src/views/404/NavigationItem/style.js.scss

```diff
@@ -0,0 +1,30 @@
+nav_item {
+    display: block;
+    padding: 10px 2.5%;
+    margin: 0 2.5%;
+    border-bottom: 1px solid #f4f4ee;
```

Comment:
Why not the new [`pasteup.js`](https://github.com/guardian/frontend/blob/master/ui/src/pasteup.js#L13)?